oteapi/models/transformationconfig.py

```diff
@@ -39,40 +39,58 @@ class TransformationConfig(GenericConfig, SecretConfig):
         description=(
             "Type of registered transformation strategy. E.g., `celery/remote`."
         ),
+        IRI="http://purl.org/dc/terms/type",  # type: ignore
     )
     name: Optional[str] = Field(
-        None, description="Human-readable name of the transformation strategy."
+        None,
+        description="Human-readable name of the transformation strategy.",
+        IRI="http://purl.org/dc/elements/1.1/title",  # type: ignore
     )
     due: Optional[datetime] = Field(
         None,
         description=(
             "Optional field to indicate a due data/time for when a transformation "
             "should finish."
         ),
+        IRI="http://purl.org/dc/terms/date",  # type: ignore
     )
     priority: Optional[ProcessPriority] = Field(
         ProcessPriority.MEDIUM,
         description="Define the process priority of the transformation execution.",
+        IRI="http://www.w3.org/ns/adms#status",  # type: ignore
     )
 
 
 class TransformationStatus(BaseModel):
     """Return from transformation status."""
 
-    id: str = Field(..., description="ID for the given transformation process.")
+    id: str = Field(
+        ...,
+        description="ID for the given transformation process.",
+        IRI="http://purl.org/dc/terms/identifier",  # type: ignore
+    )
     status: Optional[str] = Field(
-        None, description="Status for the transformation process."
+        None,
+        description="Status for the transformation process.",
+        IRI="http://www.w3.org/ns/adms#status",  # type: ignore
     )
     messages: Optional[List[str]] = Field(
-        None, description="Messages related to the transformation process."
+        None,
+        description="Messages related to the transformation process.",
+        IRI="http://purl.org/dc/terms/description",  # type: ignore
     )
     created: Optional[datetime] = Field(
         None,
         description="Time of creation for the transformation process. Given in UTC.",
+        IRI="http://purl.org/dc/terms/created",  # type: ignore
     )
     startTime: Optional[datetime] = Field(
-        None, description="Time when the transformation process started. Given in UTC."
+        None,
+        description="Time when the transformation process started. Given in UTC.",
+        IRI="http://purl.org/dc/terms/date",  # type: ignore
```

Comment:
`startTime` and `finishTime` cannot have the same IRI. However, DCAT has `dcat:startDate` and `dcat:endDate` which suitable here. 

The only issue with `dcat:startDate` and `dcat:endDate` are that they have domain `dcterms:PeriodOfTime`. Since a `TransformationStatus` describes a time period, but isn't a time periode itself, the RDF serialisation cannot be straight forward, like

```rdf
:transformation_status1 
  dcat:startTime "2024-02-29 08:45" ;
  dcat:endTime "2024-02-29 09:00" .
```

but has to has to be expressed as:
```rdf
:transformation_status1 
  dcterms:temporal [
    a dcterms:PeriodOfTime ;
    dcat:startTime "2024-02-29 08:45" ;
    dcat:endTime "2024-02-29 09:00" ;
  ] .
```